### PR TITLE
fix(Workspaces): add default workspace default database settings

### PR DESCRIPTION
**Pull request by cheikhgwane**
Fix bug that occurs when connecting to postgres and  WORKSPACE_DATABASE_ROLE default database is not set  or doesn't exist.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -35,6 +35,7 @@ x-app: &common
       - WORKSPACES_DATABASE_PASSWORD=hexa-app
       - WORKSPACES_DATABASE_HOST=db
       - WORKSPACES_DATABASE_PORT=5432
+      - WORKSPACES_DATABASE_DEFAULT_DB
```
Comment (pvanliefland) [inline docker-compose.yaml:38]: Could you add `hexa-app` as default here?

File: .env.dist [hunk 2]
```diff
@@ -6,3 +6,8 @@ EMAIL_HOST_USER=
 EMAIL_USE_TLS=
 EMAIL_HOST_PASSWORD=
 GCS_SERVICE_ACCOUNT_KEY=
+WORKSPACES_DATABASE_ROLE=
```
Comment (pvanliefland) [inline .env.dist:9]: Is this needed? All the values can be in the `docker-compose.yaml` file

**Review by pvanliefland — CHANGES_REQUESTED** [review]
2 small comments about the env variables for dev

File: .env.dist (hunk 2 above)
Comment (cheikhgwane) [reply .env.dist:9]: Not really needed.

**Review by pvanliefland — APPROVED** [review]

**Pull request merged**